### DISCUSSION TITLE
How do I add the Arabic language to the Uno in wpf and gtk?

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#8895] [Q&A]
### Current behavior

![image](https://user-images.githubusercontent.com/37013638/170561689-d592e84f-dd4e-445f-a4b4-01451d25e95a.png)
working to web and uwp
![image](https://user-images.githubusercontent.com/37013638/170561879-e75ef486-25d3-477a-8444-8d0d27031017.png)

**Comment by jeromelaban** [comment]
Thanks for your question. You're going to need to be more specific. Please create a specific sample that shows the behavior which is incorrect.

**Comment by AbdalaMask** [comment]
An example to illustrate the problem
![image](https://user-images.githubusercontent.com/37013638/170571552-e1f10f36-6ed0-42c8-8d85-1264b72c6208.png)
![image](https://user-images.githubusercontent.com/37013638/170572316-b0e85d08-63d1-420e-b120-171a2ec437c5.png)
[Test.zip](https://github.com/unoplatform/uno/files/8781869/Test.zip)

**Reply by jeromelaban** [reply]
Thanks. Can you try again with the latest Uno 3.5-dev bits ? We've changed a few things there. that may improve the rendering.

**Reply by AbdalaMask** [reply]
using 4.4.0-dev114

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/170648525-6e5f9c05-7058-430a-b342-a59594514a50.png)
The same problem

**Reply by jeromelaban** [reply]
Thanks. This is similar to https://github.com/unoplatform/uno/issues/6783. You can follow this issue for updates.
